package.json

```diff
@@ -103,7 +107,7 @@
     "graphql": "^14.7.0",
     "graphql-subscriptions": "^1.2.1",
     "graphql-tools": "^7.0.5",
-    "grpc": "^1.24.2",
```

Comment:
I have moved grpc to optional dependencies simply because it takes ages to install (lot's of C++ code that needs to be compiled), simply to speed up the `npm install` step in the Github action (via `--no-optional`). It is quite a hack. Let me know if y'all think this is a bad idea. :-)

Reply:
I think it's smart, but IMO I'd not use this, because it's really hard to understand why we are doing this + it's a misusage.

Two other options/ideas:
1. Try https://github.com/actions/cache if the installation takes too long
2. Switch to https://www.npmjs.com/package/@grpc/grpc-js

Reply:
Option (2.) does not work for us -- we instrument the "old" grpc package, thus we need that package in our dev dependencies (for the integration test). Instrumenting the "new" package `@grpc/grpc-js` is on the list I think, but not done yet, plus it would not help to get rid of the old dependency.

I'll look into (1.) 👍

Reply:
For now I have removed that commit, so grpc is back in the normal devDependencies.